src/akimbo/mixin.py

```diff
@@ -313,26 +321,40 @@ def rename(self, where, to):
         parent.fields[this] = to
         return self.to_output(ak.Array(lay))
 
-    def merge(self):
+    def pack(self):
         """Make a single complex series out of the columns of a dataframe"""
         if not self.is_dataframe(self._obj):
-            raise ValueError("Can only merge on a dataframe")
+            raise ValueError("Can only pack on a dataframe")
         out = {}
         for k in self._obj.columns:
-            # TODO: partial merge when column names are like "record.field"
+            # TODO: partial pack when column names are like "record.field"
             out[k] = self._obj[k].ak.array
         arr = ak.Array(out)
         return self.to_output(arr)
 
-    def unmerge(self):
+    def unpack(self):
         """Make dataframe out of a series of record type"""
         arr = self.array
         if not arr.fields:
             raise ValueError("Not array-of-records")
-        # TODO: partial unmerge when (some) fields are records
+        # TODO: partial unpack when (some) fields are records
         out = {k: self.to_output(arr[k]) for k in arr.fields}
         return self.dataframe_type(out)
 
+    def group_lists(self, *cols, outname="grouped"):
```

Comment:
@jpivarski : my implementation of the opposite of explode. We *could* still call this "unexplode", maybe.
```
In [1]: import polars as pl

In [2]: import akimbo.polars

In [3]: df = pl.DataFrame({"x": [1, 1, 1, 2, 1, 3, 3, 1], 'y':  [1, 1, 1, 2, 1, 3, 3, 1], 'z':  [1, 1, 1, 2, 1, 3, 3, 2]})

In [4]: df.ak.group_lists("x")
Out[4]:
shape: (3, 2)
┌─────┬─────────────────────────┐
│ x   ┆ grouped                 │
│ --- ┆ ---                     │
│ i64 ┆ list[struct[2]]         │
╞═════╪═════════════════════════╡
│ 1   ┆ [{1,1}, {1,1}, … {1,2}] │
│ 2   ┆ [{2,2}]                 │
│ 3   ┆ [{3,3}, {3,3}]          │
└─────┴─────────────────────────┘

In [5]: df.ak.group_lists("x", "y")
Out[5]:
shape: (3, 3)
┌─────┬─────┬───────────────────┐
│ x   ┆ y   ┆ grouped           │
│ --- ┆ --- ┆ ---               │
│ i64 ┆ i64 ┆ list[struct[1]]   │
╞═════╪═════╪═══════════════════╡
│ 1   ┆ 1   ┆ [{1}, {1}, … {2}] │
│ 2   ┆ 2   ┆ [{2}]             │
│ 3   ┆ 3   ┆ [{3}, {3}]        │
└─────┴─────┴───────────────────┘
```